### PR TITLE
workflows/scheduled: increase verbosity.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- output all `tar` extracted files
- output all `_site` generated files

### CODE REVIEW

**Review by Rylan12 — APPROVED** [review]
Thanks, looks good!

File: .github/workflows/scheduled.yml [hunk 1]
```diff
@@ -61,16 +56,11 @@ jobs:
...
-        env:
-          ANALYTICS_JSON_KEY: ${{ secrets.HOMEBREW_FORMULAE_BREW_SH_ANALYTICS_JSON_KEY }}
-        run: echo "$ANALYTICS_JSON_KEY" > ~/.homebrew_analytics.json
-
       - name: Update data for homebrew/core
         run: /usr/bin/rake formulae
 
       - name: Archive data
-        run: tar czvf data-core.tar.gz _data/ api/ formula/
+        run: tar czvf data-core.tar.gz _data/bottle/ _data/formula/ _data/formula_canonical.json api/bottle/ api/formula/ formula/
```
Comment (Rylan12) [inline .github/workflows/scheduled.yml:63]:
```suggestion
        run: tar czvf data-core.tar.gz _data/formula/ _data/formula_canonical.json api/formula/ formula/
```

File: .github/workflows/scheduled.yml [hunk 2]
```diff
@@ -28,18 +28,13 @@ jobs:
...
-        run: echo "$ANALYTICS_JSON_KEY" > ~/.homebrew_analytics.json
-
       - name: Update data for homebrew/cask
         run: |
           brew update-reset $(brew --repository homebrew/cask)
           /usr/bin/rake cask
 
       - name: Archive data
-        run: tar czvf data-cask.tar.gz _data/ api/ cask/
+        run: tar czvf data-cask.tar.gz _data/cask/ api/cask/ api/cask-source/ cask/
```
Comment (Rylan12) [inline .github/workflows/scheduled.yml:37]:
```suggestion
        run: tar czvf data-cask.tar.gz _data/cask/ api/cask/ cask/
```

File: .gitignore [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
 bin/
 vendor/
 .jekyll-metadata
 .homebrew_analytics.json
 .sass-cache/
 Gemfile.lock
+_data/
+api/bottle/
+api/cask/
+api/cask-source/
```
Comment (Rylan12) [inline .gitignore:13]:
```suggestion
api/cask/
```

File: Rakefile [hunk 4]
```diff
@@ -19,7 +19,7 @@ task :formulae, [:os, :tap] do |task, args|
   ENV["HOMEBREW_NO_COLOR"] = "1"
   sh "brew", "ruby", "script/generate.rb", args[:os], args[:tap]
 end
-CLOBBER.include FileList[%w[_data/formula _data/formula_canonical.json api/formula formula]]
+CLOBBER.include FileList[%w[_data/formula _data/bottle api/formula api/bottle formula _data/formula_canonical.json]]
```
Comment (Rylan12) [inline Rakefile:22]:
```suggestion
CLOBBER.include FileList[%w[_data/formula api/formula formula _data/formula_canonical.json]]
```

File: Rakefile [hunk 5]
```diff
@@ -29,7 +29,7 @@ task :cask, [:tap] do |task, args|
   ENV["HOMEBREW_NO_COLOR"] = "1"
   sh "brew", "ruby", "script/generate-cask.rb", args[:tap]
 end
-CLOBBER.include FileList[%w[_data/cask api/cask cask]]
+CLOBBER.include FileList[%w[_data/cask api/cask api/cask-source cask]]
```
Comment (Rylan12) [inline Rakefile:32]:
```suggestion
CLOBBER.include FileList[%w[_data/cask api/cask cask]]
```

File: .github/workflows/scheduled.yml (hunk 2 above)
Comment (Rylan12) [inline .github/workflows/scheduled.yml:37]:
```suggestion
        run: tar czvf data-cask.tar.gz _data/cask/ api/cask/ cask/
```

**Review by Rylan12 — COMMENTED** [review]
We can totally remove the bottle and cask-source APIs since none of them are used in Homebrew/brew anymore (unless you think others are using them and we want to just deprecate instead).

I commented some of those changes here but there are also changes needed in `script/generate.rb` and `script/generate-cask.rb`. Happy to push those changes if you'd like

**Comment by MikeMcQuaid** [comment]
> We can totally remove the bottle and cask-source APIs

Let's do that in https://github.com/Homebrew/formulae.brew.sh/pull/702. I think it needs to wait for a tag.

**Comment by Rylan12** [comment]
I think they've been removed long enough to not worry much about it, but it definitely makes more sense to remove it in there so 👍

**Review by Rylan12 — APPROVED** [review]

**Pull request merged**